tests/test_api.py

```diff
@@ -0,0 +1,18 @@
+import timereport.api as api
+
+import tests.test_data as test_data
+import os
+
+def test_factory(event):
+    # set os.environ SLACK_TOKEN
+    os.environ["SLACK_TOKEN"] = api.config['SLACK_TOKEN']
+    api.lambda_handler(event, None)
+
+
+
+
+# run test_data.add
+test_factory(test_data.add)
```

Comment:
tests in `tests` should be automated tests. If you want this manual test we should either keep it outside of repo or in another directory. Otherwise the test runner (pytest in our case) might execute this with unexpected result